tools/lsp/preview/ui.rs

```diff
@@ -16,7 +16,21 @@ export component PreviewUi inherits Window {
             checkable: true;
             clicked => { root.design_mode_changed(self.checked); }
         }
-        preview_area_container := ComponentContainer {}
+
+        scroll_area := ScrollView {
+            Rectangle {
+                width: Math.max(scroll_area.width, preview_area_container.width + 40px);
+                height: Math.max(scroll_area.height, preview_area_container.height + 40px);
+                background: Colors.red;
+
+                preview_area_container := ComponentContainer {
+                    width: 100px;
+                    height: 100px;
```

Comment:
Looks suspicious.  Wouldn't the width / height be set by the inside?

Reply:
I see why you have a binding loop. 
Maybe something like `width: self.preferred-width;`   or something like that.